prime-router/src/main/kotlin/transport/EmailTransport.kt

```diff
@@ -0,0 +1,75 @@
+package gov.cdc.prime.router.transport
+
+import com.sendgrid.*
+import com.sendgrid.helpers.mail.Mail
+import com.sendgrid.helpers.mail.objects.*
+import gov.cdc.prime.router.EmailTransportType
+import gov.cdc.prime.router.TransportType
+import gov.cdc.prime.router.azure.DatabaseAccess
+import org.thymeleaf.TemplateEngine
+import org.thymeleaf.context.Context
+import org.thymeleaf.templateresolver.StringTemplateResolver
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.Calendar
+
+class EmailTransport : ITransport {
+
+    override fun send(
+        orgName: String,
+        transportType: TransportType,
+        header: DatabaseAccess.Header,
+        contents: ByteArray
+    ): Boolean {
+
+        val emailTransport = transportType as EmailTransportType
+
+        val from = Email("qtv1@cdc.gov")
+        val subject = "COVID-19 Reporting:  Your test results are ready"
```

Comment:
I'm wondering if this can be a transport configuration?

Reply:
Now supplied as a transport config (from) with a default value of qtv1@cdc.gov (and a TODO to fix it)